client/gutenberg/editor/controller.js

```diff
@@ -121,6 +130,11 @@ export const post = async ( context, next ) => {
 
 		const Editor = initGutenberg( userId, siteSlug );
 
+		// Reset the Gutenberg state
+		use( resetGutenbergState );
```

Comment:
`use( addResetToRegistry )` in `initGutenberg` returns the registry enhanced with the `reset` method. Maybe we could do this?
```js
const { Editor, registry } = initGutenberg(); // updated return value of initGutenberg
registry.reset();
```
This way, function call is a function call and not a plugin registration 🙂